status-im/status-react
# Reply component in chat input & quoted messages redesign

fixes #13538 #13539 

### Summary

Redesign of reply component in chat input and quoted / replied messages

<img width="448" src="https://user-images.githubusercontent.com/18485527/181301866-a9551a28-7d21-411a-b97e-1c3e9c03f04e.png">

### Testing notes
- Different types of mesages

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open Status
- Open a chat
- Reply a message

- Open Status
- Open a chat
- See a replied message in a chat

status: ready
